### ISSUE TITLE
Ruffle Error on https://galaga.ee/

### ISSUE THREAD

**Issue by XFC-966** [ruffle-rs/ruffle#8286]
# Error Info
Error name: TypeError
Error message: Failed to execute 'compile' on 'WebAssembly': HTTP status code is not ok
Error stack:
```
TypeError: Failed to execute 'compile' on 'WebAssembly': HTTP status code is not ok
```

# Player Info
Allows script access: false

# Page Info
Page URL: https://galaga.ee/

# Browser Info
Useragent: Mozilla/5.0 (X11; CrOS x86_64 15054.98.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36
OS: Linux x86_64

# Ruffle Info
Version: 0.1.0
Name: nightly 2020-12-31
Channel: nightly
Built: 2020-12-31T00:48:27.350Z
Commit: ea3b6f59dfd7c528f2ae305c4dba2d351ada9302

**Comment by n0samu** [comment]
This was fixed a long time ago by #6767. The website needs to update their Ruffle version.

**Comment by XFC-966** [comment]
ok
thx
